### DISCUSSION TITLE
Table 1 Members

### DISCUSSION THREAD

**Discussion by ciarakg** [IBAHCM/RPiR#413] [Old]
Ciara Goddard : ciarakg : https://github.com/IBAHCM/ciarakg_series01.git

**Comment by IsobelAJones** [comment]
Isobel Jones: IsobelAJones: https://github.com/IBAHCM/IsobelAJones_series01.git

**Comment by Sambainbridge** [comment]
Sambainbridge: https://github.com/IBAHCM/githubSambainbridge_series0201.git

**Reply by 2700638AichiMkunde** [reply]
https://github.com/IBAHCM/aichimkunde_2021.git

**Comment by 2700638AichiMkunde** [comment]
2700638AichiMkunde: https://github.com/IBAHCM/aichimkunde_2021.git

**Comment by IsobelAJones** [comment]
Isobel Jones: IsobelAJones: https://github.com/IBAHCM/IsobelAJones_series02.git

**Reply by IsobelAJones** [reply]
@ciarakg @Sambainbridge  @2700638AichiMkunde 
Hi, I think my code is ready for checking! Hopefully it's all running smoothly.

**Comment by ciarakg** [comment]
Ciara Goddard : ciarakg : https://github.com/IBAHCM/ciarakg_series02.git

**Reply by ciarakg** [reply]
@IsobelAJones @2700638AichiMkunde @Sambainbridge Hello! Would you guys be able to check my scripts for the practical 2 series are running okay and create an issue? (link above). If you have anything that needs checking yet let me know!

**Reply by IsobelAJones** [reply]
Have done, didn't see any issues :)

**Reply by 2700638AichiMkunde** [reply]
Heys Ciara!

Its doing just fine!

**Reply by ciarakg** [reply]
Thank you both!

**Reply by Sambainbridge** [reply]
All good for me as well :)

**Reply by ciarakg** [reply]
Great. Thank you!

**Comment by Sambainbridge** [comment]
https://github.com/Sambainbridge/githubSambainbridge_series02.git


Just the link if anything needs checking!

**Reply by IsobelAJones** [reply]
Checked :)

**Reply by 2700638AichiMkunde** [reply]
Heys Sam!

Your code is working well

**Comment by 2700638AichiMkunde** [comment]
https://github.com/IBAHCM/2700638AichiMkunde_series02.git

**Reply by 2700638AichiMkunde** [reply]
Heys, may you check my code!
